roles/installer/templates/deployments/deployment.yaml.j2

```diff
@@ -84,12 +80,29 @@ spec:
               subPath: bundle-ca.crt
               readOnly: true
 {% endif %}
+{% if init_container_extra_volume_mounts -%}
+            {{ init_container_extra_volume_mounts | indent(width=12, first=True) }}
+{% endif %}
 {% if projects_persistence|bool and is_k8s|bool %}
+        - name: init-projects
+          image: '{{ _init_container_image }}'
+          imagePullPolicy: '{{ image_pull_policy }}'
+          command:
+            - /bin/sh
+            - -c
+            - |
+              chmod 775 /var/lib/awx/projects
+              chgrp 1000 /var/lib/awx/projects
+          env:
+            - name: MY_POD_NAME
+              valueFrom:
+                fieldRef:
+                  fieldPath: metadata.name
+          volumeMounts:
             - name: "{{ ansible_operator_meta.name }}-projects"
               mountPath: "/var/lib/awx/projects"
-{% endif %}
-{% if init_container_extra_volume_mounts -%}
-            {{ init_container_extra_volume_mounts | indent(width=12, first=True) }}
+          securityContext:
+            runAsUser: 0
```

Comment:
Sorry, we can't do this. OpenShift and other locked-down flavors of Kubernetes require addition privileges in order to use `securityContext`. We will need to take a look and see if there is another way of fixing this.

Reply:
Perhaps by using a bare `quay.io/centos/centos` image for this task instead of `_control_plane_ee_image` and removing this dedicated `securityContext`.

Somehow it is like reverting #1012 just for `chmod` and `chgrp`.

What do you think about ?

Reply:
Fixed in 761640bf2c3b0e135e23b97b9eaf22a95dc6e002

Tested with operator `0.30.0`

Reply:
@shanemcd is this way better?